### ISSUE TITLE
Xamarin.Forms.Maps has mispackaged the design assembly

### ISSUE THREAD

**Issue by alanmcgovern** [xamarin/Xamarin.Forms#5472]
If i examine all the nupkgs shipped by X.F we can see that every .Design.dll except for `Xamarin.Forms.Maps.Design.dll` is in the correct location.

Can we move `Xamarin.Forms.Maps.Design.dll` to a 'Design' subfolder like everything else?

```
-rw----     2.0 fat    84480 b- defN 19-Mar-05 18:49 lib/netstandard2.0/Design/Xamarin.Forms.Core.Design.dll
-rw----     2.0 fat    13824 b- defN 19-Mar-05 18:49 lib/netstandard2.0/Design/Xamarin.Forms.Xaml.Design.dll
-rw----     2.0 fat    84480 b- defN 19-Mar-05 18:49 lib/netstandard1.0/Design/Xamarin.Forms.Core.Design.dll
-rw----     2.0 fat    13824 b- defN 19-Mar-05 18:49 lib/netstandard1.0/Design/Xamarin.Forms.Xaml.Design.dll
-rw----     2.0 fat    84480 b- defN 19-Mar-05 18:49 lib/MonoAndroid90/Design/Xamarin.Forms.Core.Design.dll
-rw----     2.0 fat    13824 b- defN 19-Mar-05 18:49 lib/MonoAndroid90/Design/Xamarin.Forms.Xaml.Design.dll
-rw----     2.0 fat    84480 b- defN 19-Mar-05 18:49 lib/MonoAndroid81/Design/Xamarin.Forms.Core.Design.dll
-rw----     2.0 fat    13824 b- defN 19-Mar-05 18:49 lib/MonoAndroid81/Design/Xamarin.Forms.Xaml.Design.dll
-rw----     2.0 fat    84480 b- defN 19-Mar-05 18:49 lib/Xamarin.iOS10/Design/Xamarin.Forms.Core.Design.dll
-rw----     2.0 fat    13824 b- defN 19-Mar-05 18:49 lib/Xamarin.iOS10/Design/Xamarin.Forms.Xaml.Design.dll
-rw----     2.0 fat    84480 b- defN 19-Mar-05 18:49 lib/uap10.0/Design/Xamarin.Forms.Core.Design.dll
-rw----     2.0 fat    13824 b- defN 19-Mar-05 18:49 lib/uap10.0/Design/Xamarin.Forms.Xaml.Design.dll
-rw----     2.0 fat    84480 b- defN 19-Mar-05 18:49 lib/Xamarin.Mac/Design/Xamarin.Forms.Core.Design.dll
-rw----     2.0 fat    13824 b- defN 19-Mar-05 18:49 lib/Xamarin.Mac/Design/Xamarin.Forms.Xaml.Design.dll
-rw----     2.0 fat    84480 b- defN 19-Mar-05 18:49 lib/tizen40/Design/Xamarin.Forms.Core.Design.dll
-rw----     2.0 fat    13824 b- defN 19-Mar-05 18:49 lib/tizen40/Design/Xamarin.Forms.Xaml.Design.dll
-rw----     2.0 fat    12288 b- defN 19-Mar-05 18:49 lib/netstandard2.0/Xamarin.Forms.Maps.Design.dll
-rw----     2.0 fat    12288 b- defN 19-Mar-05 18:49 lib/netstandard1.0/Xamarin.Forms.Maps.Design.dll
```

> VS bug [#812508](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/812508), VS bug [#812681](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/812681), VS bug [#817128](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/817128), VS bug [#820632](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/820632)

**Comment by samhouts** [comment]
from @alanmcgovern 
>the design assembly has to be side-by-side for every framework
>it's missing from android, uap, tizen, ios and mac

**Comment by PureWeen** [comment]
Do we even need this project or DLL?  Does its existence even without active code cause some behavior to occur with intellisense?
https://github.com/xamarin/Duplo/pull/3277

The dll has been in the wrong folder and packaged wrong since forever and it doesn't seem like it's caused any issues.

@rmarinho @kzu   ?

**Comment by kzu** [comment]
If this assembly does not contribute any metadata, it should be completely unnecessary. It was almost surely added as boilerplate to complete later as metadata was needed (as an example, say).

**Comment by alanmcgovern** [comment]
Please don't add the previewer tag to this bug again. It's not a previewer bug and it is causing duplicate bugs to be repeatedly created. There have already been 4 duplicate bugs created against us, all of which have been reassigned back to forms.

**Comment by samhouts** [comment]
closed by #5583
